ckan_meta_tester/__init__.py

```diff
@@ -39,7 +38,9 @@ def get_pr_body(github_token: Optional[str], pr_url: Optional[str]) -> str:
 
         resp = requests.get(pr_url, headers=headers)
         if resp.ok:
-            return resp.json().get('body', '')
+            # If the PR has an empty body, 'body' is set to None, not the empty string
+            logging.warning('Pull requests should have a description with a summary of the changes')
```

Comment:
Shouldn't this warning be inside of an `if` statement that checks if the body is empty? This looks like it would be triggered every time.

Reply:
Also I'm not sure we have any magic wired up for `logging.warning` to be translated into GitHub format. Elsewhere we always emit warnings like:

```
    print(f'::warning::Warning text here')
```

Reply:
> Shouldn't this warning be inside of an if statement that checks if the body is empty? This looks like it would be triggered every time.

Oh yeah, I got played by the existing `if` there.

> Also I'm not sure we have any magic wired up for logging.warning to be translated into GitHub format. Elsewhere we always emit warnings like:

True. Oriented myself at the other `logging` statements in that file, which were all lacking the syntax.
Added a `::warning::` prefix now, but I kept it as `logging.warning()` statement instead of `print()`, should make transition easier if we ever want to do some special things with the logger.